src/dune_rules/coq/coq_rules.ml

```diff
@@ -988,12 +991,6 @@ let install_rules ~sctx ~dir s =
   | { Coq_stanza.Theory.package = Some package; buildable; _ } ->
     let loc = s.buildable.loc in
     let* mode = select_native_mode ~sctx ~dir buildable in
-    let* scope = Scope.DB.find_by_dir dir in
-    (* We force the coq scope for this DB as to fail early in case of
-       some library conflicts that would also generate conflicting
-       install rules. This is needed as now install rules less lazy
-       than the theory rules. *)
-    let _ = Scope.coq_libs scope in
```

Comment:
This forcing is an issue. It requires computing the coq scope in the install rules, which itself may force the install rules by depending on public executables.